components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/security/handler/BasicSecurityHandler.java

```diff
@@ -49,7 +41,12 @@
 
     private String name;
     private SecretResolver secretResolver;
```

Comment:
Shall we remove this variable, as we have removed the usage of it.

Reply:
done